automatminer/automl/adaptors.py

```diff
@@ -83,7 +86,7 @@ def __init__(self, **tpot_kwargs):
 
         self.from_serialized = False
         self._best_models = None
-        super(DFMLAdaptor, self).__init__()
+        super(TPOTAdaptor, self).__init__(**tpot_kwargs)
```

Comment:
Feel free to change TestAdaptorGood to not override DFMLAdaptor, since the DFMLAdaptor usage in test Adaptors should remain the same. Like we shouldn't be testing TPOTAdaptor where previously the base class was tested

Reply:
Not quite sure what you mean... Is your comment related to this line?

Also, I was wondering what's the purpose of using Python 2 `super()` syntax in automatminer if all your CI is Python 3.7? I.e. why not simply use `super()` without arguments everywhere?
```diff
- super(TPOTAdaptor, self).__init__(**tpot_kwargs)
+ super().__init__(**tpot_kwargs)
```

Reply:
using py3 super syntax is fine by me!

My comment is mostly about using any specific class in the adaptor test. The test should be for the Base adaptor class. If we need more tests for particular adaptors, they should go in that class's tests

Reply:
> using py3 super syntax is fine by me!

Cool. Would you accept a separate PR converting all `super()` calls to Py3?

Reply:
Can also make it part of this PR if you prefer.

Reply:
Yes, let's make that a separate PR though.